<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>microMouseServer</class>
 <widget class="QMainWindow" name="microMouseServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>microMouseServer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsView" name="graphics"/>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>200</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tb_status">
       <attribute name="title">
        <string>Status</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTextBrowser" name="txt_status"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tb_debug">
       <attribute name="title">
        <string>Debug</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTextBrowser" name="txt_debug"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSystem">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="menu_startRun"/>
    <addaction name="separator"/>
    <addaction name="menu_loadMaze"/>
    <addaction name="menu_saveMaze"/>
    <addaction name="separator"/>
    <addaction name="menu_connect2Mouse"/>
    <addaction name="actionConnect_to_Remote_Mouse"/>
   </widget>
   <addaction name="menuSystem"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionSave_Maze">
   <property name="text">
    <string>Save Maze</string>
   </property>
  </action>
  <action name="menu_startRun">
   <property name="text">
    <string>Start Run</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>test</string>
   </property>
  </action>
  <action name="menu_loadMaze">
   <property name="text">
    <string>Load Maze</string>
   </property>
  </action>
  <action name="menu_saveMaze">
   <property name="text">
    <string>Save Maze</string>
   </property>
  </action>
  <action name="menu_connect2Mouse">
   <property name="text">
    <string>Connect to Local Mouse</string>
   </property>
  </action>
  <action name="actionConnect_to_Remote_Mouse">
   <property name="text">
    <string>Connect to Remote Mouse</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
